concordium-node/src/configuration.rs

```diff
@@ -76,6 +76,8 @@ pub const SOFT_BAN_DURATION_SECS: u64 = 300;
 pub const MAX_PEER_NETWORKS: usize = 20;
 /// Database subdirectory name
 pub const DATABASE_SUB_DIRECTORY_NAME: &str = "database-v4";
+/// Default out-of-band catch-up file name
+pub const CATCHUP_FILE: &str = "blocks_to_import.mdb";
```

Comment:
I don't think we should be having this default. It just makes the node behaviour harder to understand. It does not improve the use of the node.